projects/aca-content/folder-rules/src/rule-details/actions/rule-action-list.ui-component.spec.ts

```diff
@@ -38,7 +39,8 @@ describe('RuleActionListUiComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [CoreTestingModule]
+      imports: [CoreTestingModule],
+      providers: [{ provide: AlfrescoApiService, useClass: AlfrescoApiServiceMock }]
```

Comment:
this line should not be needed at all, we have this configured in the CoreTestingModule

Reply:
we moved `AlfrescoApiService` from core to `@alfresco/adf-content-service` in ADF